getting-started/photometry.rst

```diff
@@ -22,107 +20,87 @@ This is the measurement strategy:
-
-This command created a ``deepCoadd_mergeDet`` dataset, which is a consistent table of sources across all filters.
+Merging the detections from the multiple bands used to produce the coadds allows later steps to use multi-band information in their processing: e.g. deblending.
+The ``mergeDetections`` sub-pipeline created a ``deepCoadd_mergeDet`` dataset, which is a consistent table of sources across all filters.
 
 .. _getting-started-tutorial-measure-coadds:
 
 Deblending and measuring source catalogs on coadds
 ==================================================
 
-Using the merged table of sources, the deblender retains all peaks and deblends any missing peaks (dropouts in that band) as PSFs. Repeating this procedure with the same master catalog across multiple coadds will generate a consistent set of child sources. Run the HSC-SDSS deblender separately in each band:
-
-.. code-block:: bash
-
-   deblendCoaddSources.py DATA --rerun coaddPhot --id filter=HSC-R
-   deblendCoaddSources.py DATA --rerun coaddPhot --id filter=HSC-I
+Seeded by the ``deepCoadd_mergeDet``, the deblender works on each detection to find the flux in each component.
+Because it has information from multiple bands, the deblender can use color information to help it work out how to separate the flux into different components.
+See the `SCARLET paper <https://arxiv.org/abs/1802.10157>`_ for further reading.
+The ``deblend`` sub-pipeline produces the ``deepCoadd_deblendedFlux`` data product.
 
-The :command:`deblendCoaddSources` command-line task produces ``deepCoadd_deblendedFlux`` datasets in the Butler data repository.
-
-Now, use the merged detection catalog to measure sources in both the ``HSC-R`` and ``HSC-I`` coadd patches.
-You can accomplish this with :command:`measureCoaddSources.py`:
-
-.. code-block:: bash
-
-   measureCoaddSources.py DATA --rerun coaddPhot --id filter=HSC-R
-
-And repeat with the ``HSC-I``-band coadd:
-
-.. code-block:: bash
-
-   measureCoaddSources.py DATA --rerun coaddPhot --id filter=HSC-I
-
-The :command:`measureCoaddSources` command-line task produces ``deepCoadd_meas`` datasets in the Butler data repository.
-Because the same merged detection catalog is used for every filter, the ``HSC-R`` and ``HSC-I``-band ``deepCoadd_meas`` tables have consistent rows.
+The ``measure`` sub-pipeline is responsible for measuring object properties on all of the deblended children produced by the deblender.
+This produces the ``deepCoadd_meas`` catalog data product with flux and shape measurement information for each object.
 You'll see how to access these tables later.
 
 .. _getting-started-tutorial-merge-coadds:
 
 Merging multi-band source catalogs from coadds
 ==============================================
 
-The previous step you created measurement catalogs for each patch in both the ``HSC-R`` and ``HSC-I`` bands.
+The previous step you created measurement catalogs for each patch, but measurements were done per band.
```

Comment:
In the previous step or the previous step created.

Reply:
I definitely think that the previous sections need restructuring a little so that they don't look like they are missing code.

Reply:
OK.  I changed the wording a little and made them subsections of the top section to make them obviously a part of that command